operator/tests/central/basic-central/10-assert.yaml

```diff
@@ -94,12 +94,6 @@ kind: RoleBinding
 metadata:
   name: central-prometheus-k8s
 ---
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: rhacs-central-auth-reader
-  namespace: kube-system
----
```

Comment:
Can you use a `TestAssert` to check the existence of this RoleBinding instead of removing the check? (applies to the 2 other RoleBindings removed as well)